ambari-server/src/main/java/org/apache/ambari/server/checks/AtlasMigrationPropertyCheck.java

```diff
@@ -58,18 +58,16 @@
     @Override
     public void perform(PrerequisiteCheck prerequisiteCheck, PrereqCheckRequest request) throws AmbariException {
       String atlasMigrationProperty = getProperty(request,"application-properties","atlas.migration.data.filename");
-      if(null == atlasMigrationProperty || StringUtils.isEmpty(atlasMigrationProperty.trim())) {
+      if(null == atlasMigrationProperty || StringUtils.isBlank(atlasMigrationProperty.trim())) {
```

Comment:
No need for the trim() or the null check when using isBlank()

Reply:
Updated in this [commit](https://github.com/apache/ambari/pull/1221/commits/16ae709b40081b5c5e4fb815093979ba6bd2f531).